examples/02_decoding/plot_miyawaki_reconstruction.py

```diff
@@ -134,17 +141,18 @@ def flatten(list_of_2d_array):
 X_test = X_test[y_test[:, 0] != -1]
 y_test = y_test[y_test[:, 0] != -1]
 
-sys.stderr.write(" Done (%.2fs).\n" % (time.time() - t0))
+sys.stderr.write(f" Done ({time.time() - t0:.2f}s).\n")
 
 ############################################################################
 # We define our prediction function
 # -----------------------------------
 sys.stderr.write("Training classifiers... \r")
 t0 = time.time()
 
+from sklearn.feature_selection import SelectKBest, f_classif
+
```

Comment:
Same here, the comment only restates what the import statement is already saying.

Reply:
done